test/e2e-tests.sh

```diff
@@ -109,6 +109,13 @@ function dump_extra_cluster_state() {
   kubectl logs $(get_app_pod controller knative-serving)
 }
 
+function publish_test_images() {
+  image_dirs="$(find $(dirname $0)/test_images -depth 1 -type d)"
```

Comment:
Should this be `-mindepth 1`? On Linux, it's an error to pass an arg to `-depth`

Reply:
In my tests on Mac and Linux, I found using `find ${REPO_ROOT_DIR}/test/test_images -mindepth 1 -maxdepth 1 -type d` works the best. @jcrossley3 @adrcunha Let me know if that works for you. `.../test/test_images/*` with just `-maxdepth` doesn't give the right paths.